tests/c20_client/test_client.py

```diff
@@ -1,17 +1,103 @@
-
 import requests_mock
-from c20_client.c20_client import do_job
+import pytest
+from c20_client.client import do_job
+from c20_client.connection_error import NoConnectionError
+
+
+def test_client_calls_decuments_endpoint_for_documents_job():
+    with requests_mock.Mocker() as mock:
+        mock.get('http://capstone.cs.moravian.edu/get_job',
+                 json={'job_type': 'documents',
+                       "url": "https://api.data.gov/json?documents=abc"})
+        mock.get('https://api.data.gov/json?documents=abc',
+                 json={'documents': 'abc'})
+        mock.post('http://capstone.cs.moravian.edu',
+                  json={})
+        do_job()
+        history = mock.request_history
+
+        assert len(history) == 3
+        assert 'capstone' in history[0].url
+        assert 'api.data.gov' in history[1].url
+        assert 'capstone' in history[2].url
 
 
-def test_client_calls_server_regulations_server():
+def test_client_calls_decument_endpoint_for_document_job():
```

Comment:
I believe this is supposed to say calls_document not calls_decument. I am guessing that is a spelling mistake. Same as the documents test above.

Reply:
I am going to fix that thanks.